nimare/meta/utils.py

```diff
@@ -119,33 +123,69 @@ def compute_kda_ma(
     )
     kernel = cube[:, np.sum(np.dot(np.diag(vox_dims), cube) ** 2, 0) ** 0.5 <= r]
 
-    all_coords = []
-    # Loop over experiments
-    for i_exp, _ in enumerate(exp_idx_uniq):
-        # Index peaks by experiment
-        curr_exp_idx = exp_idx == i_exp
-        peaks = ijks[curr_exp_idx]
+    if sum_across_studies:
+        all_values = np.zeros(shape, dtype=np.int32)
+
+        # Loop over experiments
+        for i_exp, _ in enumerate(exp_idx_uniq):
+            # Index peaks by experiment
+            curr_exp_idx = exp_idx == i_exp
+            peaks = ijks[curr_exp_idx]
+
+            sphere_coords = _convolve_sphere(kernel, peaks, np.array(shape))
+
+            # Go from sparse to dense
+            study_values = np.zeros(shape, dtype=np.int32)
+
+            if sum_overlap:
+                sphere_coords = unique_rows(sphere_coords)
```

Comment:
```suggestion
                study_values[sphere_coords[:, 0], sphere_coords[:, 1], sphere_coords[:, 2]] += value
```

would this be the equivalent for the sum_overlap option?

Reply:
Ah yes, good catch. Although no tests for it since there aren't any kernels that yet need both sum across studies and sum overlap.

Reply:
Actually, I'm nots ure this work. 

I suggest we don't support `sum_overlap` since it's not needed by anything